Lib/fontTools/ttLib/tables/ttProgram.py

```diff
@@ -1067,10 +1069,25 @@ class Program(object):
     def __init__(self):
         pass
 
+    @property
+    def assembly(self):
```

Comment:
since you typed the other one, how about `def assembly(self) --> List[str]:`

Reply:
I added some more annotations.